tools/tracing/lock_contended.bt

```diff
@@ -0,0 +1,11 @@
+uprobe:$MMTK:_ZN3std3sys4unix5locks11futex_mutex5Mutex14lock_contended17he467d89501d7157aE {
```

Comment:
The last part `17he467d89501d7157aE` is specific to the build.  In my local build, the same function is suffixed with `17h2a372fcb57338e4dE`.  Fortunately, `bpftrace` supports wildcard in function names.

```suggestion
uprobe:$MMTK:_ZN3std3sys4unix5locks11futex_mutex5Mutex14lock_contended* {
```